### ISSUE TITLE
EpoxyProcessorException: resolveDataBindingClassesAndWriteJava failed

### ISSUE THREAD

**Issue by iamMehedi** [airbnb/epoxy#1081]
Build fails with the following error:
```
Task :app:kaptFullDebugKotlin
error: EpoxyProcessorException: resolveDataBindingClassesAndWriteJava failed
  com.airbnb.epoxy.processor.EpoxyProcessorException: resolveDataBindingClassesAndWriteJava failed
  	at com.airbnb.epoxy.processor.Logger.logError(Logger.kt:63)
  	at com.airbnb.epoxy.processor.Asyncable$DefaultImpls.awaitAndLog(Asyncable.kt:136)
  	at com.airbnb.epoxy.processor.Asyncable$awaitAndLog$1.invokeSuspend(Asyncable.kt)
  	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
  	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:56)
  	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
  	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:738)
  	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
  	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
  Caused by: java.lang.IllegalStateException: Type mirror does not exist for androidx.databinding.ViewDataBinding
  	at com.airbnb.epoxy.processor.Utils.areParamsTheSame(Utils.kt:325)
  	at com.airbnb.epoxy.processor.Utils.getMethodOnClass(Utils.kt:290)
  	at com.airbnb.epoxy.processor.Utils.implementsMethod(Utils.kt:266)
  	at com.airbnb.epoxy.processor.GeneratedModelWriter.generateDataBindingMethodsIfNeeded(GeneratedModelWriter.kt:1191)
  	at com.airbnb.epoxy.processor.GeneratedModelWriter.generateClassForModel(GeneratedModelWriter.kt:150)
  	at com.airbnb.epoxy.processor.GeneratedModelWriter.generateClassForModel$default(GeneratedModelWriter.kt:121)
  	at com.airbnb.epoxy.processor.DataBindingProcessor$resolveDataBindingClassesAndWriteJava$2.invokeSuspend(DataBindingProcessor.kt:117)
  	at com.airbnb.epoxy.processor.DataBindingProcessor$resolveDataBindingClassesAndWriteJava$2.invoke(DataBindingProcessor.kt)
  	at com.airbnb.epoxy.processor.Asyncable$filter$2$invokeSuspend$$inlined$map$lambda$1.invokeSuspend(Asyncable.kt:77)
  	... 6 moreerror: EpoxyProcessorException: resolveDataBindingClassesAndWriteJava failed
  com.airbnb.epoxy.processor.EpoxyProcessorException: resolveDataBindingClassesAndWriteJava failed
  	at com.airbnb.epoxy.processor.Logger.logError(Logger.kt:63)
  	at com.airbnb.epoxy.processor.Asyncable$DefaultImpls.awaitAndLog(Asyncable.kt:136)
  	at com.airbnb.epoxy.processor.Asyncable$awaitAndLog$1.invokeSuspend(Asyncable.kt)
  	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
  	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:56)
  	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
  	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:738)
  	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
  	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
  Caused by: java.lang.IllegalStateException: Type mirror does not exist for androidx.databinding.ViewDataBinding
  	at com.airbnb.epoxy.processor.Utils.areParamsTheSame(Utils.kt:325)
  	at com.airbnb.epoxy.processor.Utils.getMethodOnClass(Utils.kt:290)
  	at com.airbnb.epoxy.processor.Utils.implementsMethod(Utils.kt:266)
  	at com.airbnb.epoxy.processor.GeneratedModelWriter.generateDataBindingMethodsIfNeeded(GeneratedModelWriter.kt:1191)
  	at com.airbnb.epoxy.processor.GeneratedModelWriter.generateClassForModel(GeneratedModelWriter.kt:150)
  	at com.airbnb.epoxy.processor.GeneratedModelWriter.generateClassForModel$default(GeneratedModelWriter.kt:121)
  	at com.airbnb.epoxy.processor.DataBindingProcessor$resolveDataBindingClassesAndWriteJava$2.invokeSuspend(DataBindingProcessor.kt:117)
  	at com.airbnb.epoxy.processor.DataBindingProcessor$resolveDataBindingClassesAndWriteJava$2.invoke(DataBindingProcessor.kt)
  	at com.airbnb.epoxy.processor.Asyncable$filter$2$invokeSuspend$$inlined$map$lambda$1.invokeSuspend(Asyncable.kt:77)
  	... 6 moreerror: EpoxyProcessorException: resolveDataBindingClassesAndWriteJava failed
  com.airbnb.epoxy.processor.EpoxyProcessorException: resolveDataBindingClassesAndWriteJava failed
  	at com.airbnb.epoxy.processor.Logger.logError(Logger.kt:63)
  	at com.airbnb.epoxy.processor.Asyncable$DefaultImpls.awaitAndLog(Asyncable.kt:136)
  	at com.airbnb.epoxy.processor.Asyncable$awaitAndLog$1.invokeSuspend(Asyncable.kt)
  	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
  	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:56)
  	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
  	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:738)
  	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
  	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
  Caused by: java.lang.IllegalStateException: Type mirror does not exist for androidx.databinding.ViewDataBinding
  	at com.airbnb.epoxy.processor.Utils.areParamsTheSame(Utils.kt:325)
  	at com.airbnb.epoxy.processor.Utils.getMethodOnClass(Utils.kt:290)
  	at com.airbnb.epoxy.processor.Utils.implementsMethod(Utils.kt:266)
  	at com.airbnb.epoxy.processor.GeneratedModelWriter.generateDataBindingMethodsIfNeeded(GeneratedModelWriter.kt:1191)
  	at com.airbnb.epoxy.processor.GeneratedModelWriter.generateClassForModel(GeneratedModelWriter.kt:150)
  	at com.airbnb.epoxy.processor.GeneratedModelWriter.generateClassForModel$default(GeneratedModelWriter.kt:121)
  	at com.airbnb.epoxy.processor.DataBindingProcessor$resolveDataBindingClassesAndWriteJava$2.invokeSuspend(DataBindingProcessor.kt:117)
  	at com.airbnb.epoxy.processor.DataBindingProcessor$resolveDataBindingClassesAndWriteJava$2.invoke(DataBindingProcessor.kt)
  	at com.airbnb.epoxy.processor.Asyncable$filter$2$invokeSuspend$$inlined$map$lambda$1.invokeSuspend(Asyncable.kt:77)
  	... 6 moreerror: EpoxyProcessorException: resolveDataBindingClassesAndWriteJava failed
  com.airbnb.epoxy.processor.EpoxyProcessorException: resolveDataBindingClassesAndWriteJava failed
  	at com.airbnb.epoxy.processor.Logger.logError(Logger.kt:63)
  	at com.airbnb.epoxy.processor.Asyncable$DefaultImpls.awaitAndLog(Asyncable.kt:136)
  	at com.airbnb.epoxy.processor.Asyncable$awaitAndLog$1.invokeSuspend(Asyncable.kt)
  	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
  	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:56)
  	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
  	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:738)
  	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
  	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
  Caused by: java.lang.IllegalStateException: Type mirror does not exist for androidx.databinding.ViewDataBinding
  	at com.airbnb.epoxy.processor.Utils.areParamsTheSame(Utils.kt:325)
  	at com.airbnb.epoxy.processor.Utils.getMethodOnClass(Utils.kt:290)
  	at com.airbnb.epoxy.processor.Utils.implementsMethod(Utils.kt:266)
  	at com.airbnb.epoxy.processor.GeneratedModelWriter.generateDataBindingMethodsIfNeeded(GeneratedModelWriter.kt:1191)
  	at com.airbnb.epoxy.processor.GeneratedModelWriter.generateClassForModel(GeneratedModelWriter.kt:150)
  	at com.airbnb.epoxy.processor.GeneratedModelWriter.generateClassForModel$default(GeneratedModelWriter.kt:121)
  	at com.airbnb.epoxy.processor.DataBindingProcessor$resolveDataBindingClassesAndWriteJava$2.invokeSuspend(DataBindingProcessor.kt:117)
  	at com.airbnb.epoxy.processor.DataBindingProcessor$resolveDataBindingClassesAndWriteJava$2.invoke(DataBindingProcessor.kt)
  	at com.airbnb.epoxy.processor.Asyncable$filter$2$invokeSuspend$$inlined$map$lambda$1.invokeSuspend(Asyncable.kt:77)
  	... 6 more
> Task :app:kaptFullDebugKotlin FAILED
```
Seeing this error mostly on the CI. And sometimes  it gets resolved if I run the task again.

**Comment by fanmingyi** [comment]
Me do too.

**Comment by shah-nishant** [comment]
We are facing similar issue. Were you able to find a fix?

**Comment by RageshAntony** [comment]
I am also . Why no reply

**Comment by JKhanh** [comment]
I'm having this issue too when i try to add databinding pattern

**Comment by johnpaulmckean** [comment]
Guess I'll chime in too.

**Comment by szeidner** [comment]
I'm seeing this in Android Studio Canary quite a bit as well.

**Comment by elihart** [comment]
Can everyone reporting this specify whether you have `enableParallelEpoxyProcessing` enabled? Does disabling `enableParallelEpoxyProcessing` fix the issue?

**Comment by szeidner** [comment]
Setting `enableParallelEpoxyProcessing` to "false" does seem to fix the issue for me.

**Comment by supakorn0812** [comment]
I and my team facing the same problem when I update the library version 4.5.0, but after we update the android studio 4.2 RC 1 or higher. everything works as normal.

**Comment by elihart** [comment]
I believe that the update to Java 11 in android studio 4.2 causes different javac behavior in the processor that breaks `enableParallelEpoxyProcessing`.

With KSP around the corner I don't anticipate investing time to fix it for 4.2 so if you see this problem just disable `enableParallelEpoxyProcessing`.

**Comment by PedroMoniz** [comment]
Having the same issue.

**Comment by elihart** [comment]
This should be fixed with 5.0.0, which removes parallel processing
